docs/build/circuit-construction.ipynb

```diff
@@ -0,0 +1,486 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "bc54d7bc-b2d8-4b30-9b0b-05689e07a463",
+   "metadata": {},
+   "source": [
+    "# Construct a circuit"
```

Comment:
Could you rename the page to "Circuit Construction"?

Reply:
Best practices encourage making titles as active as possible (Visualize a circuit, Simulate with..., Configure with..., etc.) so I vote to keep it `Construct a circuit` for consistency, unless it's a specific request or no other way around it.... thoughts @javabster ?

Reply:
hahaha I apologize for the intrusive Zs instead of Ss 😆  I'm totally fine with pluralizing circuits in these titles. We just need to remember to change them both on the page themselves and the toc.